[Galina] Our Menus page - HTML & CSS workshop

Galina86 opened the pull request:
Our Menus page code is ready to be reviewed

Commit:
add background and new font

Commit:
add semantic tags

Commit:
style header links

Commit:
style recipe cards

Commit:
style footer

Commit:
style hall of fame card

Commit:
add align-items

Commit:
formatting

Commit:
more formatting

Commit:
make mobile view

tlerias commented on index.html:
```diff
@@ -1,117 +1,286 @@
 
-            <div>
-                <img src="https://img.hellofresh.com/c_fill,f_auto,fl_lossy,h_432,q_auto,w_768/hellofresh_s3/image/kikkoman-tropical-teriyaki-pork-bowls-e8103c70.jpg">
-                <span>Kikkoman® Tropical Teriyaki Pork Bowls</span>
-                <span>with Kiwi Salsa, Lime Crema & Jasmine Rice</span>
-                <div><span>35 min</span></div>
-            </div>
+  <link rel="preconnect" href="https://fonts.gstatic.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" />
+  <link
+    href="https://fonts.googleapis.com/css2?family=Anton&family=Merriweather:wght@300&display=swap"
+    rel="stylesheet"
+  />
```
all of this should be inside the `<head></head>`

tlerias commented on index.html:
```diff
@@ -1,117 +1,286 @@
+          class="recipe-card-image"
+          style="max-width: 100%; height: auto"
+        />
+        <h2>Kikkoman® Tropical Teriyaki Pork Bowls</h2>
+        <span class="recipe-description"
+          >with Kiwi Salsa, Lime Crema & Jasmine Rice</span
+        >
+        <div class="prep-time"><span>35 min</span></div>
```
you can add a div around these and give it some padding to take the text off the edges without affecting the image

tlerias replied:
(and for all of the other recipes)

tlerias commented on style.css:
```diff
@@ -0,0 +1,235 @@
+* {
```
completely valid CSS but we don't recommend directly targeting HTML tags because as applications get larger it can create bugs and unwanted "side effects"

tlerias commented:
Just left some minor comments but overall really great work! The responsiveness is amazing, and not easy to do for a beginner.

Commit:
address comments and fix mobile view

Galina86 commented:
> Just left some minor comments but overall really great work! The responsiveness is amazing, and not easy to do for a beginner.

Thank you! Addressed all the comments!

tdss22 commented on style.css:
```diff
@@ -0,0 +1,229 @@
+  font-size: 32px;
+}
+
+h2 {
```
not a huge deal, but i would avoid styling elements like this. better to add a class. you will see why when we have a larger app 🙂

tdss22 commented:
@Galina86 great job with this! 🎉
